uniprot/README.md

```diff
@@ -0,0 +1,17 @@
+# Uniprot Import Script
```

Comment:
The README file content could be moved to the beginning of the `uniprot_importer.py`
This way the info to run the script is in the same file as the script.